_posts/2018-05-21-event-store-4.1.1-hotfix-1-release.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Event Store 4.1.1 Hotfix 1"
+author: "Shaan Nobee"
+layout: blog-post
+---
+
+A hotfix for Event Store 4.1.1 has been released to fix the following issue:
```

Comment:
@shaan1337 A great writing style to introduce, and I can go into more detail on this, is 'active voice' instead of 'passive voice', this basically means making the actor and subject of the sentence clear, so in this case.

> We have releases a hot fix for Event Store 4.1.1 to fix the following issue: